internal/cfgplugins/bgp.go

```diff
@@ -622,3 +624,112 @@ func ConfigureBGPNeighbor(t *testing.T, dut *ondatra.DUTDevice, ni *oc.NetworkIn
 	nAfiSafi.GetOrCreateAddPaths().Receive = ygot.Bool(sendReceivePaths)
 	nAfiSafi.GetOrCreateAddPaths().Send = ygot.Bool(sendReceivePaths)
 }
+
+// ConfigureDUTBGP configures BGP on the DUT using OpenConfig.
+func ConfigureDUTBGP(t *testing.T, dut *ondatra.DUTDevice, dutAS, ate1AS, ate2AS, ate3AS, ate4AS, ate5AS, ecmpMaxPath uint32,
```

Comment:
This function should probably only configure the Global BGP parameters.

The appendBGPNeighbor should be exported and called by the _test.go.  This makes the code more reusable (so a test can decide how many neighbors are needed)